submissions/adeyemiJuwon/HTMLforms/login.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```

Comment:
Remove this login page then. It is not suppose to exist yet. @Samju200

Reply:
Okay
Done